docs/digger/installation/known-issues.asciidoc

```diff
@@ -5,36 +5,30 @@
 
 ==== HTTP proxy with macOS Jenkins node
 
-When configuring Jenkins to work behind a HTTP proxy, there is a https://issues.jenkins-ci.org/browse/JENKINS-31464[limitation/bug] in the way slave nodes use the Git plugins. This is because the slaves inherit proxy information from the master.
+When configuring Jenkins to run behind a HTTP proxy, there is a https://issues.jenkins-ci.org/browse/JENKINS-31464[limitation/bug] in the way slave nodes use the Git plugins. This is because the slaves inherit proxy information from the master.
```

Comment:
lets make the link here open a new tab

Reply:
nice! It mildly infuriates me as well if my current webpage is suddenly replaced by linked.
resolved